docs/application/dotnet/guides/user-interface/nui/text.md

```diff
@@ -312,10 +312,169 @@ The following markup elements are currently supported:
+    ```
+
+-   `<u>`
+
+    Sets the underlined values for the characters inside the element.
+
+    The following attributes are supported:
+
+    -   `color`: The color of underline.
+    -   `height`: The height of underline.
+    -   `type`: The type of underline. The supported attribute values are `solid`, `dashed` and `double`.
+    -   `dash-gap`: The gap in pixels between the dashes of the dashed underline. Only valid when "DASHED" underline type is used.
+    -   `dash-width`: The width in pixels of the dashes of the dashed underline. Only valid when "DASHED" underline type is used.
+
+    The following example shows how to apply it:
+
+    ```xaml
+    <base:TextLabel x:Name="txt" EnableMarkup="True"
+        Text="<u color='green' height='2.0f' type='dashed' dash-gap='2.0f' dash-width='3.0f'>Hello world</u>"/>
+    ```
+
+-   `<s>`
+
+    Sets the strikethrough values for the characters inside the element.
+
+    The following attributes are supported:
+
+    -   `color`: The color of strikethrough.
+    -   `height`: The height of strikethrough.
+
+    The following example shows how to apply it:
+
+    ```xaml
+    <base:TextLabel x:Name="txt" EnableMarkup="True"
+        Text="<s color='green' height='2.0f' >Hello world</s>"/>
+    ```
+
+-   `<char-spacing>`
+
+    Sets the character spacing values for the characters inside the element.
+
+    Use the `value` attribute to define the spaces between characters in Pixels.
+
+    A positive value will make the characters far apart (expanded) and a negative value will bring them closer (condensed).
+
+    The following example shows how to apply it:
+
+    ```xaml
+    <base:TextLabel x:Name="txt" EnableMarkup="True"
+        Text="<char-spacing value='5.0f'>Hello world</char-spacing>"/>
+    ```
+
+-   `<p>`
+
+    Use paragraph tag to set many styles on paragraph's level for the lines inside the element.
+
+    The following attributes are supported:
+
+    -   `align`: Use the `align` attribute to define the horizontal alignment of paragraph. The supported attribute values are `begin`, `center` and `end` .
+    -   `rel-line-height`: The relative height of the line (a factor that will be multiplied by text height).
+
+    The following example shows how to apply alignment:
+
+    ```xaml
+    <base:TextLabel x:Name="txt" EnableMarkup="True"
+        Text="text outside<p align='end'>Paragraph end</p>text outside<p align='center'>Paragraph center</p>text outside<p align='begin' >Paragraph begin</p>"/>
+    ```
+
+    The following example shows how to apply relative height of line:
+
+    ```xaml
+    <base:TextLabel x:Name="txt" EnableMarkup="True"
+        Text="<p>line 1</p><p rel-line-height=0.5>line\n2</p>line 3<p rel-line-height=3>line\n4</p>line 5"/>
+    ```
+
+-   `<item>`
+
+    Defines an embedded item within the text.
+
+    The following attributes are supported:
+
+    -   `url`: The url path of the image.
```

Comment:
URL should be capitalized here. Please look into it.
```suggestion
    -   `url`: The URL path of the image.
```

Reply:
Done